ansible/roles/devtoolset-7/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: devtoolset-7 | Install devtoolset-7
+  yum:
+    name: "{{ item }}"
+    state: latest
```

Comment:
This should be `present` upgrades can be handled separately.

Reply:
I've fixed this now.